bazel_rules/cc_rules.bzl

```diff
@@ -0,0 +1,6 @@
+def cc_binary_with_headers(name, srcs, hdrs, includes=[]):
```

Comment:
Binaries do not generally have headers, switch to cc_llibrary or remove the `.h` file